assignment/scripts/function-practice.js

```diff
@@ -15,30 +15,40 @@ console.log('Test - should say "Hello World!"', hello());
 // 2. Function to return an personalized hello, using the `name` argument.
 //    for example 'Hello, Jo!', or 'Hello, Stacy!'
 function helloName( name ) {
-  return;
+  return `Hello, ${name}!`;
 }
 // Remember to call the function to test
+console.log(`Test - helloName should say "Hello, Steve!": ${helloName('Steve')}`);
 
 
 // 3. Function to add two numbers together & return the result
-function addNumbers( firstNumber ) {
-  // return firstNumber + secondNumber;
+function addNumbers( firstNumber, secondNumber ) {
+  return firstNumber + secondNumber;
 }
 
+console.log(`Test - should output 5: ${addNumbers(2, 3)}`);
+console.log(addNumbers(5, 5));
+console.log(addNumbers(7, 10));
+console.log(addNumbers(1, 0));
 
-// 4. Function to multiply three numbers & return the result
-function multiplyThree( ){
 
+// 4. Function to multiply three numbers & return the result
+function multiplyThree(x, y, z){
+  return x*y*z;
```

Comment:
Spaces around operators help make code more readable.

```suggestion
  return x * y * z;
```